src/cgame/cg_rocket_events.cpp

```diff
@@ -55,6 +55,11 @@ static void CG_Rocket_EventShow()
 	Rocket_DocumentAction( CG_Argv( 1 ), "show" );
 }
 
+static void CG_Rocket_EventBlurAll()
+{
+	Rocket_DocumentAction( CG_Argv( 1 ), "blurall" );
```

Comment:
blurall doesn't take an argument. The empty string is usually used for the first parameter

Reply:
I followed the convention in the rest of the file I guess.

Fixed.